tests/testthat/test-font-objects.R

```diff
@@ -0,0 +1,142 @@
+library(htmltools)
```

Comment:
Can't really test this file if we're offline. 

```suggestion
skip_if_offline()

library(htmltools)
```

IDK if you'd like to do another test that checks that google fonts is at least responding. Like check a fixed url to see that the status code is a 200 or download.file doesn't throw an error.